src/Psl/Xml/format_errors.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Xml;
+
+use LibXMLError;
+use Psl\Arr;
+use Psl\Str;
+
+/**
+ * This function can be used to format multiple LibXMLErrors.
+ *
+ * @psalm-param list<LibXMLError> $errors
+ */
+function format_errors(array $errors): string
+{
+    return Str\join(
+        Arr\values(
```

Comment:
Any clue why psalm doesn't know that the map actually returns a list?
I had to add this `values` hack since I couldn't find it out... The types in `Arr\map` seem valid.

Reply:
`Arr\map` is `Arr\map<Tk as array-key, Tv, Ts>(array<Tk, Tv>, (function(Tk): Ts)): array<Tk, Ts>`, psalm can't know from this signature that the returned array is in the same order, all it knows is that the key type is the same, and since `list<T>` is `array<int, T>` ( but ordered ), the return type would be `array<int, T>`, again, psalm can't verify that it's ordered.